src/guides/v2.3/extension-dev-guide/indexing.md

```diff
@@ -99,9 +99,13 @@ The Magento indexing mechanism uses the status value in reindex triggering proce
 Reindexing can be performed in two modes:
 
 *  Update on Save - index tables are updated immediately after the dictionary data is changed.
+
+{:.bs-callout-info}
+**Update on Save** Indexers must be handled by custom code (plugins, events or any other working approach) to trigger re-index logic when involved entities are saved/deleted etc., and the indexer is configured as Update On Save.
```

Comment:
I would suggest omitting the last part since we are talking about "Update on Save" from the very beginning of the sentence 

```suggestion
**Update on Save** Indexers must be handled by custom code (plugins, events or any other working approach) to trigger re-index logic when involved entities are saved/deleted etc.
```